### ISSUE TITLE
Public communication channel for project

### ISSUE THREAD

**Issue by lukecoy** [mozillascience/software-discovery-dashboard#36]
To foster a sense of community / project accessibility, we should have an IRC or a Slack or Etherpad or something like that. I prefer Etherpad because its super simple to just pop in without providing any information besides your name. Lets discuss :)

**Comment by varzaman** [comment]
Etherpad is a good idea. It has chat and persistent text which can become useful which the other services don't really have(if you don't count chat history).

**Comment by mok4ry** [comment]
I don't know that Etherpad is the best communication tool. Personally, I've only used Etherpad alongside some other communication medium, like Skype or even in person. Maybe it's because the chat is emphasized very little. I'd prefer Slack or IRC. Slack is the more friendly option, in my opinion. Is it possible to have an open Slack that anyone can join?

**Comment by nmanaud** [comment]
Hi guys. We had a similar choice to make for our OpenPlanetary community, and you opted for Slack (despite the limitation of the free account), although there is an open-source version it: http://www.mattermost.org. Something you may also want to look it.

**Comment by varzaman** [comment]
I agree with Matt about using IRC.
It is free, open source, it is already widely known for developers. 
Etherpad is good for meetings, but IRC would facilitate ongoing, and quick communications better I would think.

**Comment by mok4ry** [comment]
+1. Those unfamiliar with IRC are likely not code contributors and will probably contribute mostly by taking part in useful discussions on GitHub. IRC is a very helpful tool for code contributors by nature of the generally fast response time it elicits.

That being said, we should have a "joining IRC" walkthrough in the `README` that would make even those unfamiliar with IRC comfortable connecting and participating in the conversation that occurs there.

**Comment by varzaman** [comment]
We need to get it set up first haha.

I wonder if Mozilla would give us a channel on their IRC.

@acabunoc ?

**Comment by lukecoy** [comment]
IRC +1

**Comment by abbycabs** [comment]
what channel do you want?

**Comment by abbycabs** [comment]
you can create one by joining it :)

**Comment by varzaman** [comment]
Oh, good to know, thanks!

**Comment by varzaman** [comment]
IRC channel created(check the readme for information).
